includes/reports/pipeline_by_rep.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (params) => {
+  return publish("report_pipeline_by_rep", {
+    description: "Pipeline by Rep",
+    columns: {
+    // DIMENSIONS
+
+
+    // MEASURES
+    
+    },
+    assertions: {
+      //
+    },
+    ...params.defaultConfig
+  }).query(ctx => `
+
+    with opportunitites as (
+        select *
+        from ${ctx.ref("salesforce_opportunities")}
+    
+    ),
+    
+    users as (
+        select *
+        from ${ctx.ref("salesforce_users")}
+    
+    ),
+    
+    pipeline_by_rep as (
+        select 
+            -- dimensions
+            u.name             as rep_name,
+            o.stage_name       as stage_name,       
+            o.close_date,
+
+            -- measures
+            sum(amount)        as opportunity_amount
+        from opportunities o 
+        join users u
+        on u.user_id = o.owner_id
+        and o.stage_name in ('Evaluation', 'Contract Negotiation', 'Closed Won') 
```

Comment:
It might be worth pulling out some of the stage names into a js file as a list of constants